src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -113,38 +73,134 @@ with some exceptions for quantitative data.
 
 ![Cell Alignment](figma:patterns-tables-basic-formatting-cell-alignment.png)
 
-<table>
-  <thead>
-    <tr>
-      <th width="70%">Cell format</th>
-      <th>Alignment</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>1. Text</td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>2. Categorical data that uses numbers (for example, dates, phone numbers)</td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>3. Quantitative data</td>
-      <td>Right align</td>
-    </tr>
-    <tr>
-      <td>
-        4. <a href="/components/tags">Tags</a> or <a href="/components/avatar">Avatars</a>
-      </td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>5. Icons</td>
-      <td>Center align</td>
-    </tr>
-  </tbody>
-</table>
+| <b>Cellc content</b>                                                                                        | <b>Alignment</b> |
+| ----------------------------------------------------------------------------------------------------------- | ---------------- |
+| 1. Text                                                                                                     | Left align       |
+| 2. Categorical data that uses numbers (for example, [dates](/content/grammar#date-and-time), phone numbers) | Left align       |
+| 3. Quantitative data                                                                                        | Right align      |
+| 4. [Tags](/components/tags) or [Avatars](/components/avatar)                                                | Left align       |
+| 5. Icons                                                                                                    | Center align     |
+
+## Avatars, icons, and tags
+
+### Avatars
+
+[Avatars](/components/avatar) help represent a person or brand in tables. They can display icons or images.
+
+### Size
+
+Avatar sizes for tables include default and small.
+
+![Small avatars in a Default table](figma:patterns-tables-basic-formatting-small-avatars-default-table.png)
+
+- Use a small Avatar in a default table when more density is needed when displaying information
+  about many users at once
+- Default size tables with an Avatar have a height of 52px with 10px of top and bottom padding and 12px of
```

Comment:
Is the `52px` height more of what the consumer should expect? I ask because often these cell heights are fluid, e.g. avatar plus padding only happens to be `52px` here, but some content could make it even taller.

Just a thought. My main concern is providing information that may not always be true, and so would confuse consumers implementing their own tables.

Reply:
that's a great point. i believe that is the case, tables are fluid and other content could expand the rows, so it won't always be 52px. @siennagonzalez how could we reword this?

Reply:
Changed the copy a bit to include "starting height" to imply it wont always be that size.